dronecode_sdk/__init__.py

```diff
@@ -51,3 +51,31 @@ def connect(*args, **kwargs):
         globals()[plugin](plugin_manager)
 
     return plugin_manager
+
+
+import atexit
+import os
+import subprocess
+import sys
+
+from dronecode_sdk import bin
+from time import sleep
+
+if sys.version_info >= (3,7):
+    from importlib.resources import path
+else:
+    from importlib_resources import path
+
+def start_mavlink():
+    """
+    Starts the gRPC server in a subprocess, listening on localhost:50051
+    """
+    with path(bin, 'backend_bin') as backend:
+        p = subprocess.Popen(os.fspath(backend), shell=False, stdout=subprocess.DEVNULL, stderr=subprocess.DEVNULL)
+
+    sleep(2)
```

Comment:
Why this?

Reply:
:sweat_smile: because I want to make sure the server is started before `start_mavlink()` returns. In Swift we use a library instead of running an executable in a subprocess, so we get a callback when the server is ready. Here we don't.

Improvements could be one of:
* use the library instead of the executable, but it implies wrapping the C++ interface (with Cython or something like that?)
* check the grpc server status instead of a dumb `sleep`

Reply:
Ok, we can improve it later. For now I'd add a comment describing why.